mirrord/console/src/main.rs

```diff
@@ -0,0 +1,111 @@
+mod protocol;
+
+use std::any::type_name;
+
+use axum::{
+    extract::ws::{Message, WebSocket, WebSocketUpgrade},
+    response::Response,
+    routing::get,
+    Router,
+};
+use protocol::Hello;
+use tracing::{error, info, metadata::LevelFilter};
+use tracing_subscriber::{fmt::format::FmtSpan, prelude::*};
+
+struct WsWrapper {
```

Comment:
```suggestion
struct WebSocketWrapper {
```
Looks like the convention is to spell it out